### ISSUE TITLE
RFC 6902 Patch Format Support

### ISSUE THREAD

**Issue by Miserlou** [olafura/json_diff_ex#38]
Hi there!

This is an excellent library, very fast! However, it does not seem to output to RFC 6902 format. It'd be wonderfully useful for web and live applications if this was a feature!

**Comment by Miserlou** [comment]
ex: https://github.com/benjamine/jsondiffpatch/blob/master/src/formatters/jsonpatch.js

**Comment by olafura** [comment]
@Miserlou sounds like a nice feature to add, have used JSON Patch, is a bit verbose for transports but way more understandable.

**Comment by Miserlou** [comment]
Yeah! It's fatter on the wire, but since it's standardized it should be supported in more places, which makes it handy.

BTW, I'm using your library downstream here: https://github.com/Miserlou/live_json It's using another library for JSON-Patch support right now, but uses yours as the default behavior since it's much faster.
